app/models/public_body.rb

```diff
@@ -129,6 +129,17 @@ class Translation
   #
   # [1] http://git.io/vIetK
   class Version
+    before_save :copy_translated_attributes
+
+    def copy_translated_attributes
+      public_body.attributes.each do |name, value|
+        if self.public_body.translated?(name) &&
+            !public_body.non_versioned_columns.include?(name)
+          self.send("#{name}=", value)
+        end
+      end
+    end
```

Comment:
failing because of broken specs? if so, explain what specs were broken and why this fixes.

If no failing specs, then we probably need some extra coverage here.